front-end/src/components/Header.js

```diff
@@ -3,7 +3,7 @@ import 'react-sticky-header/styles.css'
 import StickyHeader from 'react-sticky-header'
 import './Header.css'
 import Hamburger from './Hamburger'
-import logo from '../images/logo.png'
+import logo from '../images/newLogo.png'
```

Comment:
newLogo seems to be an old version of the new logo; i'd suggest sticking with logo.png